app/views/pages/_topics.html.erb

```diff
@@ -1,69 +1,64 @@
+      </li>
+      <li>
+        <h3 class="govuk-heading-s dgu-topics__heading"><%= link_to t('.crime_and_justice_label'), search_path(filters: { topic: 'Crime and justice' }), class: 'govuk-link' %></h3>
+        <p class="govuk-body"><%= t('.crime_and_justice_description') %></p>
+      </li>
+      <li>
+        <h3 class="govuk-heading-s dgu-topics__heading"><%= link_to t('.defence_label'), search_path(filters: { topic: 'Defence' }), class: 'govuk-link' %></h3>
+        <p class="govuk-body"><%= t('.defence_description') %></p>
+      </li>
+      <li>
+        <h3 class="govuk-heading-s dgu-topics__heading"><%= link_to t('.education_label'), search_path(filters: { topic: 'Education' }), class: 'govuk-link' %></h3>
+        <p class="govuk-body"><%= t('.education_description') %></p>
+      </li>
+    </ul>
   </div>
 
-  <div class="column-one-third">
-    <div>
-      <ol>
+  <div class="govuk-grid-column-one-third">
+      <ul>
         <li>
-          <h2><%= link_to t('.environment_label'), search_path(filters: { topic: 'Environment' }), class: 'govuk-link' %></h2>
-          <p><%= t('.environment_description') %></p>
+          <h3 class="govuk-heading-s dgu-topics__heading"><%= link_to t('.environment_label'), search_path(filters: { topic: 'Environment' }), class: 'govuk-link' %></h3>
+          <p class="govuk-body"><%= t('.environment_description') %></p>
         </li>
         <li>
-          <h2><%= link_to t('.government_label'), search_path(filters: { topic: 'Government' }), class: 'govuk-link' %></h2>
-          <p><%= t('.government_description') %></p>
+          <h3 class="govuk-heading-s dgu-topics__heading"><%= link_to t('.government_label'), search_path(filters: { topic: 'Government' }), class: 'govuk-link' %></h3>
+          <p class="govuk-body"><%= t('.government_description') %></p>
         </li>
         <li>
-          <h2><%= link_to t('.government_spending_label'), search_path(filters: { topic: 'Government spending' }), class: 'govuk-link' %></h2>
-          <p><%= t('.government_spending_description') %></p>
+          <h3 class="govuk-heading-s dgu-topics__heading"><%= link_to t('.government_spending_label'), search_path(filters: { topic: 'Government spending' }), class: 'govuk-link' %></h3>
+          <p class="govuk-body"><%= t('.government_spending_description') %></p>
         </li>
         <li>
-          <h2><%= link_to t('.health_label'), search_path(filters: { topic: 'Health' }), class: 'govuk-link' %></h2>
-          <p><%= t('.health_description') %></p>
+          <h3 class="govuk-heading-s dgu-topics__heading"><%= link_to t('.health_label'), search_path(filters: { topic: 'Health' }), class: 'govuk-link' %></h3>
+          <p class="govuk-body"><%= t('.health_description') %></p>
         </li>
-      </ol>
-    </div>
+      </ul>
   </div>
-  <div class="column-one-third">
-    <div>
+  <div class="govuk-grid-column-one-third">
       <ol>
```

Comment:
The previous two columns have unordered lists rather than ordered lists - should this column be the same?
```suggestion
      <ul class="govuk-list">
```